server/models.py

```diff
@@ -958,7 +959,7 @@ def create(cid, enrollment_info, role=STUDENT_ROLE):
 
 class Message(Model):
     __tablename__ = 'message'
-    __table_args__ = {'mysql_row_format': 'COMPRESSED'}
+    __table_args__ = {'mysql_row_format': os.getenv('DB_ROW_FORMAT', 'COMPRESSED')}
```

Comment:
It would be nice to store `DB_ROW_FORMAT` in the `settings/*.py` files and use `app.config` here rather than `os.getenv`. This looks to be related: https://stackoverflow.com/questions/49409691/flask-sqlalchemy-acess-different-schema

Reply:
+1 on storing on settings/*.py

Reply:
Could you provide some guidance on how we'd achieve this? In the `models.py` file we don't have access to the `app`. I tried accessing the app via `flask.current_app` (see [current-app.patch.txt](https://github.com/Cal-CS-61A-Staff/ok/files/1974530/current-app.patch.txt)) but that gives me an error:

![image](https://user-images.githubusercontent.com/1086421/39629927-a891948e-4fa5-11e8-9515-b1aed0869199.png)